.travis.yml

```diff
@@ -297,9 +297,9 @@ jobs:
           on:
             all_branches: true
         - provider: script
-            script: bash scripts/travis/copy_share_image_to_dockerHub.sh
-            on:
-              all_branches: true
+          script: bash scripts/travis/copy_share_image_to_dockerHub.sh
```

Comment:
@PunchedChimera Was this an intentional change (formatting only) ?

Reply:
Yes, the incorrect indentation actually caused the travis.yml to not be parsable by the build agents. This fixes it.